sql/jdbc/src/main/java/org/apache/spark/sql/jdbc/Utils.java

```diff
@@ -68,7 +68,7 @@
     // Note on client side parameter naming convention:
     // Prefer using a shorter camelCase param name instead of using the same name as the
     // corresponding
-    // SparkServer2 config.
+    // SparkThriftServer2 config.
```

Comment:
nit: if we're renaming everywhere, then maybe drop the "2" and make it just SparkThriftServer?

Reply:
> nit: if we're renaming everywhere, then maybe drop the "2" and make it just SparkThriftServer?

yea， support it, change later